docs/general/emulator-support-and-issues.md

```diff
@@ -276,6 +276,7 @@ This page focuses on supported emulators. For extensive notes on unsupported emu
 
 - ✅ libretro core: **Flycast**
   - Disable threaded rendering to properly use save states.
+- ✅ Standalone emulator: **[Flycast](https://github.com/flyinghead/flycast)**
```

Comment:
gamecube has this:

- `Must use version 2407-68 or newer.`

so probably should i add

- Must use version  2.4 or newer.

ref: https://github.com/flyinghead/flycast/releases/tag/v2.4

Reply:
@Jamiras

Reply:
2.4 is correct, but as best as I can tell, Flycast is currently sending a version number of 1.0:
```
Flycast/1.0
```
This should probably be fixed in Flycast itself.